scripts/apps/templates/directives/TemplatesDirective.js

```diff
@@ -314,9 +315,9 @@ export function TemplatesDirective(gettext, notify, api, templates, modal, desks
 
                 if (deskNames) {
                     return notify.error(
-                        gettext('This is a default template of the following desk(s):') +
-                        ` ${deskNames}. ` +
-                        gettext('Please change the default templates first.'));
+                        gettext('This is a default template of the following desk(s): {{deskNames}}. ' +
```

Comment:
Looks like concatenation is just fine:
#: scripts/apps/templates/directives/TemplatesDirective.js:318
msgid "This is a default template of the following desk(s): {{deskNames}}. Please change the default templates first."
msgstr ""

Reply:
I'm surprised it works, but I would still like to have nothing, but plain strings without any kind of logic inside `gettext` calls. It would be a cleaner and more portable solution if we wanted to switch a library for extraction or write our own.